fh_fablib/__init__.py

```diff
@@ -710,7 +716,7 @@ def deploy(ctx, fast=False):
     check(ctx)
     run(ctx, f"git push origin {config.branch}")
     if not fast:
-        run(ctx, "NODE_ENV=production npx webpack -p --bail")
+        build_frontend(ctx)
```

Comment:
Das reicht leider nicht. `build_frontend` referenziert nicht den Task aus der Collection, sondern direkt die Methode oben.

(D.h. auch, wenn Du zB `check` überschreibst, wird hier im `deploy` Task trotzdem der Standard-Check ausgeführt.)

Es ist in diesem Fall leider im Moment das einfachste und einzig mögliche, den ganzen `deploy` Task im projektspezifischen Fabfile nachzubauen. Leider unterstützt fabric 2 `execute()` noch nicht (https://github.com/pyinvoke/invoke/issues/170)